app/main.py

```diff
@@ -1,3 +1,18 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    def __init__(self, name, description, weeks):
+        self.name = name
+        self.description = description
+        self.weeks = weeks
+
+    @staticmethod
+    def days_to_weeks(days):
+        import math
+        return math.ceil(days / 7)
```

Comment:
I dont sure about import here. I think better:
from math import ceil
At the first line
We can ask this moment tomorrow)